addon/mixins/relationship-support-model.js

```diff
@@ -2,10 +2,28 @@
 
 import { RELATIONSHIP_ADAPTER_OPTION } from '@bagaaravel/ember-data-extensions/config'
 import getRelationshipDescriptor from '@bagaaravel/ember-data-extensions/utils/get-relationship-descriptor'
-import { assert } from '@ember/debug'
+import { assert, deprecate } from '@ember/debug'
 import Mixin from '@ember/object/mixin'
 
 export default Mixin.create({
```

Comment:
Maybe you can refactor the mixin code to use the util internally? I know it's going to be deleted though.

Reply:
Yeah, weird I forgot about this, f*ck was I thinking. It'll save some lines of code as well I think. Coming up!

Reply:
Done.